assets/templates/partials/calendar/items/list.tmpl

```diff
@@ -4,12 +4,22 @@
   tag designed for the purpose:
     <em class="ons-highlight">keyword</em>
 */}}
+{{ $currentPage := .Pagination.CurrentPage }}
+{{ $itemsPerPage := .Pagination.Limit }}
+{{ $totalSearchPosition := multiply (subtract $currentPage 1) $itemsPerPage }}
```

Comment:
Can this logic be abstracted to `mapper.go` and proved by unit tests?

Reply:
This should be working now.